src/test/datascience/notebook.functional.test.ts

```
@@ -1467,14 +1467,14 @@ plt.show()`,
             runTest('Current directory', async () => {
                 const rootFolder = ioc.get<IWorkspaceService>(IWorkspaceService).rootPath!;
                 const escapedPath = `'${rootFolder.replace(/\\/g, '\\\\')}'`;
-                addMockData(`import os\nprint(os.getcwd())`, escapedPath);
+                addMockData(`import os\nos.getcwd()`, escapedPath);
                 const notebook = await notebookProvider.getOrCreateNotebook({
                     identity: getDefaultInteractiveIdentity(),
                     resource: Uri.file(path.join(rootFolder, 'foo.ipynb'))
                 });
 
                 assert.ok(notebook, 'did not create notebook');
-                await verifySimple(notebook, `import os\nprint(os.getcwd())`, escapedPath, true);
+                await verifySimple(notebook, `import os\nos.getcwd()`, escapedPath, true);
```

print in this test makes the output streaming output as opposed to text plain. Our verifySimple is built for checking text plain, so just converted it to remove the print. These issues can slip in since we set our own mock data above, so the error is only noticeable in the full nightly tests, not the PR tests.